curvefs/src/client/fuse_volume_client.cpp

```diff
@@ -205,9 +205,12 @@ CURVEFS_ERROR FuseVolumeClient::FuseOpWrite(fuse_req_t req, fuse_ino_t ino,
         inode.set_length(off + *wSize);
     }
 
-    uint64_t nowTime = TimeUtility::GetTimeofDaySec();
-    inode.set_ctime(nowTime);
-    inode.set_atime(nowTime);
+    struct timespec now;
+    clock_gettime(CLOCK_REALTIME, &now);
+    inode.set_mtime(now.tv_sec);
+    inode.set_mtime_ns(now.tv_nsec);
+    inode.set_ctime(now.tv_sec);
+    inode.set_ctime_ns(now.tv_nsec);
```

Comment:
old code is ctime and atime
new code is mtime and ctime

Reply:
Write modify  mtime is right.